doc/api/tls.md

```diff
@@ -821,6 +821,28 @@ The `callback` function, if specified, will be added as a listener for the
 
 `tls.connect()` returns a [`tls.TLSSocket`][] object.
 
+To upgrade an existing instance of `net.Socket` to a 
```

Comment:
this is documented above, does it really need an example? was the existing description of the `socket: ` option somehow deficient?